File: solutions/Word.py
#==========================
#Method-1: Using .islower()
#==========================

s = input()
lower, upper = 0, 0

for x in s:
  if x.islower(): lower += 1
  else: upper += 1

if lower >= upper: print(s.lower())
else: print(s.upper())

#===========================
#Method-2: Using ord() 
#===========================

s = input()
lower, upper = 0, 0

for x in s:
  if 0 <= ord(x) - ord('a') < 26: lower +=1
  else: upper += 1

if lower >= upper: print(s.lower())
else: print(s.upper())

File: solutions/Word.c
#include <stdio.h>
#include <string.h>
#include <ctype.h> 

int main() {
    char s[105];
    scanf("%s", s);
    int n = strlen(s);
    
    int lower = 0, upper = 0;
    
    for (int i = 0; i < n; i++) {
        if (islower(s[i])) lower++;
        else upper++;
    }
    
    if (lower >= upper) {
        for (int i = 0; i < n; i++) {
            printf("%c", tolower(s[i]));
        }
    }
    else {
        for (int i = 0; i < n; i++) {
            printf("%c", toupper(s[i]));
        }
    }
    return(0);
}